.gitignore

```diff
@@ -3,6 +3,7 @@ xcuserdata
 *.xccheckout
 ResourceApp/rswift
 R.generated.swift
+R.watchOS.generated.swift
```

Comment:
Why do we generate a seperate file for watchOS, is this necessary? The watch is a seperate target if I'm correct, just like tvOS and iOS are separate. I think it would be possible (and nicer) to generate a `R.generated.swift` file for the watchOS target, it has also needs it's own setup in that case.

But correct me if I'm wrong and this isn't possible.